src/core/entities/MRStats.js

```diff
@@ -0,0 +1,20 @@
+import { MRDivEntity } from '../MRDivEntity';
+
+// TODO: JSDoc
+
+export class MRStats extends MRDivEntity {

```

Comment:
MRStats should be an `MREntity` instead of `MRDivEntity`

think of `MRDivEntity` items to be (except for MRModel) the UI elements - that is theyre the ones that are situated on a panel almost as if that panel is a website in 3D space

Since we want MRStats to be its own object regardless of a panel and to act for the app as a whole - `MREntity` is better